src/environments/keys-config.ts

```diff
@@ -3,12 +3,12 @@
  * https://firebase.google.com/docs/web/setup
  */
 export const FirebaseConfig = {
-  apiKey: '',
-  authDomain: '',
-  databaseURL: '',
-  projectId: '',
-  storageBucket: '',
-  messagingSenderId: ''
+  apiKey: 'AIzaSyCo7Sd8MSlfc9G5ScyG7_XOzxp9uEZWzAg',
```

Comment:
this should not be tracked, please remove and rebase so that this change is not included in the git history